src/libtiled/wangset.cpp

```diff
@@ -134,6 +187,71 @@ void WangId::flipVertically()
     rotate(2);
 }
 
+WangIdVariationIterator::WangIdVariationIterator(int edgeColors, int cornerColors, WangId wangId)
```

Comment:
In that case, it will be better for it to fail immediately, so maybe you don't need that if...